articles/libraries/chemistry/concepts/quantum-dynamics.md

```diff
@@ -10,7 +10,7 @@ uid: microsoft.quantum.chemistry.concepts.quantumdynamics
 
 # Quantum Dynamics
 
-Quantum mechanics is largely the study of quantum dynamics, which seeks to understand how an initial quantum state (in Dirac notation, see [conceptual docs](xref:microsoft.quantum.concepts.dirac-notation) on quantum computing for more info) $|\psi(0)\rangle$ varies over time.  Specifically, given this initial condition a quantum state, an evolution time and a specification of the quantum dynamical system, a quantum state $|\psi(t)\rangle$ is sought.  Before proceeding to explain quantum dynamics, it is useful to take a step back and think about classical dynamics since it provides insights into how quantum mechanics is really not that different from classical dynamics.
+Quantum mechanics is largely the study of quantum dynamics, which seeks to understand how an initial quantum state $|\psi(0)\rangle$ varies over time (see [conceptual docs](xref:microsoft.quantum.concepts.dirac) on quantum computing for more info on Dirac notation).  Specifically, given this initial condition a quantum state, an evolution time and a specification of the quantum dynamical system, a quantum state $|\psi(t)\rangle$ is sought.  Before proceeding to explain quantum dynamics, it is useful to take a step back and think about classical dynamics since it provides insights into how quantum mechanics is really not that different from classical dynamics.
```

Comment:
```suggestion
Quantum mechanics is largely the study of quantum dynamics, which seeks to understand how an initial quantum state $\ket{\psi(0)}$ varies over time (see the [conceptual docs](xref:microsoft.quantum.concepts.dirac) on quantum computing for more info on Dirac notation).  Specifically, given this initial condition a quantum state, an evolution time and a specification of the quantum dynamical system, a quantum state $|\psi(t)\rangle$ is sought.  Before proceeding to explain quantum dynamics, it is useful to take a step back and think about classical dynamics since it provides insights into how quantum mechanics is really not that different from classical dynamics.
```